.github/workflows/test-build.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test and Build
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+jobs:
+  test-build:
+    name: test-build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: go
+
+      - name: Install kubebuilder
+        run: |
+          set -x
+          os=$(go env GOOS)
+          arch=$(go env GOARCH)
+          curl -Ls https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
```

Comment:
Consider making the kubebuilder version a parameter?

Reply:
Done